hypothesis-python/src/hypothesis/internal/conjecture/engine.py

```diff
@@ -8,6 +8,8 @@
 # v. 2.0. If a copy of the MPL was not distributed with this file, You can
 # obtain one at https://mozilla.org/MPL/2.0/.
 
+from __future__ import annotations
+
```

Comment:
Fairly strong preference to avoid this, sorry, since it sets up [PEP-563](https://peps.python.org/pep-0563/) semantics but that has been superseded by [PEP-649](https://peps.python.org/pep-0649/) (in Python 3.13).  Yeah, it's kinda messy, but we'll all be glad of it in a few years 😅